src/browser/connection/gateway.ts

```diff
@@ -184,6 +186,15 @@ export default class BrowserConnectionGateway {
         }
     }
 
+    private static _onCloseWindowRequest (req: IncomingMessage, res: ServerResponse, connection: BrowserConnection): void {
+        if (BrowserConnectionGateway._ensureConnectionReady(res, connection)) {
+            connection.provider.closeBrowserChildWindow(connection.id)
```

Comment:
Should we catch errors of the `connection.provider.closeBrowserChildWindow` method here?